### PR TITLE
Added docker-compose service

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: roles/docker-compose-service/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The role is driven by a list of services defined in a list called `services` where each item is a dict which describes various service related configurations.
+
+
+The service list can be defined in the `hostvars` or `group_vars` ansible directory structure
+
+```yaml
+#sample service metadata
+
+service:
```
Comment (carinadigital) [inline roles/docker-compose-service/README.md]: Will this not collide with ansible's `service` ?

Comment (carinadigital) [inline roles/docker-compose-service/README.md]: should this be `services`?

If it's `service` it'll collide with ansible `service` resource

File: roles/docker-compose-service/templates/task.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+[
```
Comment (carinadigital) [inline roles/docker-compose-service/templates/task.json]: Where is this template used?

File: roles/docker-compose-service/README.md (hunk 1 above)
Comment (brettminnie) [reply roles/docker-compose-service/README.md]: let me confirm w/meta-data

File: roles/docker-compose-service/templates/task.json (hunk 2 above)
Comment (brettminnie) [reply roles/docker-compose-service/templates/task.json]: Ask @tim-mcarthur-moj  :)

File: roles/docker-compose-service/README.md (hunk 1 above)
Comment (brettminnie) [reply roles/docker-compose-service/README.md]: you are correct, it is services, I'll fix the readme

File: roles/docker-compose-service/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+The role is driven by a list of services defined in a list called `services` where each item is a dict which describes various service related configurations.
+
+
+The service list can be defined in the `hostvars` or `group_vars` ansible directory structure
+
+```yaml
+#sample service metadata
+
+services:
```
Comment (carinadigital) [inline roles/docker-compose-service/README.md]: Can I be an ass and ask for a better key name than `services`? Something so we know it's docker-compose

Comment (Azulinho) [reply roles/docker-compose-service/README.md]: docker-compose-services

Comment (Azulinho) [reply roles/docker-compose-service/README.md]: services-in-docker-compose

Comment (Azulinho) [reply roles/docker-compose-service/README.md]: services-from-docker-compose

File: roles/docker-compose-service/templates/docker-compose-service-systemd.j2 [hunk 4]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Startup script for docker-compose-{{ item.name }}
+
+[Service]
+WorkingDirectory=/etc/docker-compose/{{ item.name }}
+
+ExecStart       = /usr/local/bin/docker-compose -p {{ item.name }} -f /etc/docker-compose/{{ item.name }}/docker-compose.yml up -d
+ExecStartPost   = /usr/sbin /conntrack -D -p udp
```
Comment (Azulinho) [inline roles/docker-compose-service/templates/docker-compose-service-systemd.j2]: "/usr/sbin /conntrack -D -p udp" < extra space in there

**Review by Azulinho — CHANGES_REQUESTED** [review]
/usr/sbin' '/conntrack -D -p udp

**Pull request merged**